lib/controllers/foods-controller.js

```diff
@@ -26,4 +26,39 @@ const FoodsRouter = require('express').Router();
       });
   });
 
+  FoodsRouter.delete('/:id', (request, response) => {
+    Food.find(request.params.id)
+    .then((data) => {
+      if (data) {
+        Food.deleteFood(data.id)
+        .then( data => {
+          response.sendStatus(201);
+        })
+        .catch( error => {
+          response.status(500).send({ error: error.message });
+        });
+      } else {
+        response.status(404).send( { error: 'Resource not found' } )
+      }
+    })
+    .catch((error) => {
+      response.status(500).send( { error: error.message } )
+    })
+  })
+
+    FoodsRouter.put('/:id', (request, response) =>{
+      Food.find(request.params.id)
+        .then((data) => {
+          if (data) {
+            Food.updateFood(data.id)
```

Comment:
@scmountain @maxglassie i believe this `Food.updateFood(data.id)` will need a `then` attached for it to execute. i know this is in your `put` request, but it could be causing problems